Observer/PredispatchAdminActionControllerObserver.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * Taxjar_SalesTax
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @category   Taxjar
+ * @package    Taxjar_SalesTax
+ * @copyright  Copyright (c) 2017 TaxJar. TaxJar is a trademark of TPS Unlimited, Inc. (http://www.taxjar.com)
+ * @license    http://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
+ */
+
+
+
+namespace Taxjar\SalesTax\Observer;
+
+use Magento\Framework\Event\ObserverInterface;
+
+class PredispatchAdminActionControllerObserver implements ObserverInterface
```

Comment:
Can we give this observer a clearer name like AdminNotifications similar to M1? Let's rename the class and file itself, and also events.xml